### ISSUE TITLE
How to place remove button inside formly-group

### ISSUE THREAD

**Issue by n3xus** [formly-js/ngx-formly#840]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[X] support request
```

**Current behavior**
Remove button is placed below formly-group

**Expected behavior**
To place button inside formly-group next to inputs

https://formly-js.github.io/ngx-formly/examples/advanced/repeating-section
On repeating section example Remove button is placed below row. I'm wondering if there is a way to place it next to input boxes.

**Comment by aitboudad** [comment]
adding `<ng-content></ng-content>` in https://github.com/formly-js/ngx-formly/blob/master/src/core/src/components/formly.group.ts would solve the issue, can you make a PR ?

**Comment by n3xus** [comment]
Thanks @aitboudad That did the trick. Created Pull Request

**Comment by aitboudad** [comment]
published under `2.0.0-rc.48`.
